### DISCUSSION TITLE
Base Branch and Default branch is same .How to raise PR?

### DISCUSSION THREAD

**Discussion by anuradha2022-2022** [community/community#37084] [Other Feature Feedback, Questions, & Ideas]
I have created a branch called 'cart ' , which is working as default branch as well as base branch . Now, to create a PR ,I am unable to process further.
[https://github.com/anuradha2022-2022/Shopping-Cart/compare](url)
Please do help me out !

**Comment by paulius-valiunas** [comment]
you don't need a pull request, all your code is already in the main branch. You can rename the branch here if you want to: https://github.com/anuradha2022-2022/Shopping-Cart/branches

**Reply by anuradha2022-2022** [reply]
okay .Thanks!

**Comment by ossanna16** [comment]
Hi there @anuradha2022-2022 and welcome to our community! Thank you for asking a great question 🙂
